setup.cfg

```diff
@@ -1,10 +1,11 @@
-[isort]
-combine_as_imports = true
-default_section = THIRDPARTY
-include_trailing_comma = true
-line_length = 79
-multi_line_output = 5
-not_skip = __init__.py
-known_standard_library = marshal
-known_first_party = voc
-sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+[flake8]
+# https://flake8.readthedocs.org/en/latest/
+exclude=settings.py,wsgi.py,manage.py,*/migrations/*
+max-complexity = 10
+max-line-length = 119
+[metadata]
+description-file = README.md
+[pep8]
```

Comment:
`pep8` was renamed into `pycodestyle`
So.. maybe use `pycodestyle` instead? :smiley:
http://pycodestyle.readthedocs.io/en/latest/
I recall reading a discussion where it was proposed that `pep8` be removed from pypi, but I don't know what's the outcome of that.